src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write your code here
+const population = document.getElementsByClassName('population');
```

Comment:
```suggestion
const population = [...document.getElementsByClassName('population')];
```
And instead of creating a new array `text`, iterating with `for` loop, use reduce directly on `population`